jsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@components/*": ["components/*"]
+    "@components/*": ["components/*"]
```

Comment:
Indentations help us with understanding the hierarchy. `paths` is parent for `@components/*` right?